### DISCUSSION TITLE
Доступ определённого пользователя к привязанной к нему строке таблицы

### DISCUSSION THREAD

**Discussion by gusin-itas** [totumonline/totum-ru-issues-and-discussions#432] [Как это сделать?] [answered]
Есть таблица регистрации персональных данных пользователей. 

После первичного занесения пользователя (логин/пароль) в системную таблицу "пользователи" нужно дать созданному пользователю доступ только к его строке в таблице регистрации данных для самостоятельного занесения оставшихся персональных данных.

Каким образом это сделать?

**Comment by erpic-ru** [comment, marked as answer]
1. Лучше персональные данные и пользователей делать в разных таблицах, но это дело хозяйское
2. [Делается через префильтр с кодом ](https://ru.docs.totum.online/prefilter#type-prefilter-2)
что-то вроде:
`=: if (condition: $#nr = "id нужной роли"; then: $#nu)`
Видимость и редактирование префильтра для этой роли нужно отключить
